Dockerfile

```diff
@@ -32,7 +28,7 @@ COPY public /srv/ttn-lorawan/public
 
 COPY --from=builder /srv/ttn-lorawan/lorawan-frequency-plans /srv/ttn-lorawan/lorawan-frequency-plans
 COPY --from=builder /srv/ttn-lorawan/lorawan-webhook-templates /srv/ttn-lorawan/lorawan-webhook-templates
-COPY --from=builder /srv/ttn-lorawan/lorawan-devices-index /srv/ttn-lorawan/lorawan-devices-index
+COPY data/lorawan-devices-index /srv/ttn-lorawan/lorawan-devices-index
 RUN chown thethings:thethings -R /srv/ttn-lorawan/lorawan-devices-index
```

Comment:
@neoaggelos probably out of scope for this PR, but is this still necessary now that we open the index in read-only mode?